docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.8'
 services:
   postgres:
-    image: postgres
+    image: postgres:15
```

Comment:
can we use version 16 here?

Reply:
> can we use version 16 here?

yes we can but, version 15 is  stable due to it's maturity in market whereas version 16 is just released in sept 2023. But, if I may ask why do you insist on using version 16?

Reply:
@dishak v16 should be stable also v16 is much faster from what they say. Let's stick to it.

Reply:
@siinghd sure.. updated it.

Reply:
@siinghd , Are the PR changes satisfactory? should I close the PR if all good?